src/sentry/web/client_config.py

```diff
@@ -181,7 +181,7 @@ def get_client_config(request=None):
         "urlPrefix": options.get("system.url-prefix"),
         "version": version_info,
         "features": enabled_features,
-        "distPrefix": get_frontend_app_asset_url("sentry", ""),
+        "distPrefix": get_frontend_dist_prefix(),
```

Comment:
How come this isn't using `get_frontend_app_asset_url()` ?

Reply:
because `""` does not start with `entrypoints/` -- this was originally a kludge and now it's more straightforward